app/src/main/java/org/dieschnittstelle/mobile/android/skeleton/TaskListViewMapActivity.java

```diff
@@ -0,0 +1,82 @@
+package org.dieschnittstelle.mobile.android.skeleton;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.lifecycle.ViewModelProvider;
+
+import android.os.Bundle;
+import android.util.Log;
+
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.LatLngBounds;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+import org.dieschnittstelle.mobile.android.skeleton.model.Task;
+import org.dieschnittstelle.mobile.android.skeleton.viewmodel.TaskListViewModel;
+
+import java.util.List;
+
+public class TaskListViewMapActivity extends AppCompatActivity implements OnMapReadyCallback, GoogleMap.OnMapClickListener {
+
+    public static final String TASK_LIST_VIEW_MAP_KEY = "taskListViewMapObject";
+    private TaskListViewModel viewModel;
+    private List<Task> tasks;
+    private GoogleMap map;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_task_list_view_map);
+        viewModel = new ViewModelProvider(this).get(TaskListViewModel.class);
+        tasks = (List<Task>) getIntent().getSerializableExtra(TASK_LIST_VIEW_MAP_KEY);
```

Comment:
hier soll eine Überprüfung geben, ob `tasks != null` ist

```suggestion
        tasks = (List<Task>) getIntent().getSerializableExtra(TASK_LIST_VIEW_MAP_KEY);
        if (tasks == null) {
            tasks = new ArrayList<>();
        }
```